CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
```

Comment:
Nit: now that `master` has been renamed as `main`, let's update the changelog heading as well.